### PR TITLE
HEEDLS-218 Tutorials and assessments list front end

### PR DESCRIPTION
**Pull request by kwunkiuma**
Add hardcoded cards to section page, logic will need to be added later on to conditionally display the diagnostic and post learning assessment cards.

Screenshot of the page with hardcoded values:
image

### CODE REVIEW

File: DigitalLearningSolutions.Web/Views/LearningMenu/Section/Section.cshtml [hunk 1]
```diff
@@ -28,5 +28,55 @@
       <path d="M8.5 12c0-.3.1-.5.3-.7l5-5c.4-.4 1-.4 1.4 0s.4 1 0 1.4L10.9 12l4.3 4.3c.4.4.4 1 0 1.4s-1 .4-1.4 0l-5-5c-.2-.2-.3-.4-.3-.7z"></path>
     </svg>
     Return to course menu
-  </a> 
+  </a>
+</div>
+
+<div class="nhsuk-card nhsuk-u-margin-bottom-3 learning-menu-card">
```
Comment (FridaTveit) [inline DigitalLearningSolutions.Web/Views/LearningMenu/Section/Section.cshtml:34]: Worth extracting the assessment cards to partials as well I think 🙂

File: DigitalLearningSolutions.Web/Views/LearningMenu/_TutorialCard.cshtml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      </div>
+      <div class="nhsuk-grid-column-one-quarter">
+        <h3 class="nhsuk-heading-xs nhsuk-u-secondary-text-color">
+          Not started
+        </h3>
+      </div>
+    </div>
+    <div class="nhsuk-grid-row">
+      <div class="nhsuk-grid-column-full">
+        <a class="nhsuk-button nhsuk-u-margin-bottom-0 nhsuk-u-margin-top-0" asp-action="Tutorial" asp-route-tutorialId="1071" asp-route-sectionId="232" asp-route-customisationId="9865">
```
Comment (FridaTveit) [inline DigitalLearningSolutions.Web/Views/LearningMenu/_TutorialCard.cshtml:17]: Is this a real action which exists at the moment? If the tutorial page hasn't been created yet then it's better for this button to do nothing for now, and whoever merges last out of you and @benjimarshall can make it use the correct action 🙂

**Review by FridaTveit — COMMENTED** [review]
A few comments, otherwise looks good 🙂

File: DigitalLearningSolutions.Web/Views/LearningMenu/_TutorialCard.cshtml (hunk 2 above)
Comment (kwunkiuma) [reply DigitalLearningSolutions.Web/Views/LearningMenu/_TutorialCard.cshtml:17]: The action doesn't exist yet so I'll remove that.

File: DigitalLearningSolutions.Web/Views/LearningMenu/Section/Section.cshtml [hunk 3]
```diff
@@ -10,6 +11,8 @@
   ViewData["CustomisationId"] = 18438;
   ViewData["HeaderPath"] = $"{Configuration["AppRootPath"]}/LearningMenu/18438";
   ViewData["HeaderPathName"] = "CMS Demonstration Course";
+  var diagnosticAssessmentStatus = "Not attempted";
+  var postLearningAssessmentStatus = "Not attempted";
```
Comment (FridaTveit) [inline DigitalLearningSolutions.Web/Views/LearningMenu/Section/Section.cshtml:15]: We shouldn't be setting any variables like this here 🙂

File: DigitalLearningSolutions.Web/Views/LearningMenu/Section/Section.cshtml [hunk 4]
```diff
@@ -31,52 +34,8 @@
...
-    <div class="nhsuk-grid-row">
-      <div class="nhsuk-grid-column-full">
-        <a class="nhsuk-button nhsuk-u-margin-bottom-0 nhsuk-u-margin-top-0">
-          Launch assessment
-        </a>
-      </div>
-    </div>
-  </div>
-</div>
+<partial name="_DiagnosticAssessmentCard" model="diagnosticAssessmentStatus" />
```
Comment (FridaTveit) [inline DigitalLearningSolutions.Web/Views/LearningMenu/Section/Section.cshtml:37]: No need to pass in any model here as everything is hardcoded for now. Once we have the data from the backend we'll be passing in the real model 🙂

**Review by FridaTveit — COMMENTED** [review]
A few comments, otherwise looks good 🙂

File: DigitalLearningSolutions.Web/Views/LearningMenu/Section/Section.cshtml (hunk 4 above)
Comment (kwunkiuma) [reply DigitalLearningSolutions.Web/Views/LearningMenu/Section/Section.cshtml:37]: Ah my bad, that's from my original approach of using the same partial view.

File: DigitalLearningSolutions.Web/Views/LearningMenu/_DiagnosticAssessmentCard.cshtml [hunk 5]
```diff
@@ -0,0 +1,26 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.LearningMenu
+@model string
```
Comment (FridaTveit) [inline DigitalLearningSolutions.Web/Views/LearningMenu/_DiagnosticAssessmentCard.cshtml:2]: Should not have model

File: DigitalLearningSolutions.Web/Views/LearningMenu/_DiagnosticAssessmentCard.cshtml [hunk 6]
```diff
@@ -0,0 +1,26 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.LearningMenu
```
Comment (FridaTveit) [inline DigitalLearningSolutions.Web/Views/LearningMenu/_DiagnosticAssessmentCard.cshtml:1]: Is this using statement necessary?

**Review by FridaTveit — COMMENTED** [review]
A few minor things, otherwise looks good 🙂

**Review by FridaTveit — APPROVED** [review]
Looks good 🙂

**Pull request merged**